### ISSUE TITLE
Complete accessibility testing for iterated accordion component

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#2287]
## What

Complete accessibility testing for the iterated accordion component.

## Why

GOV.UK Accessibility team have done some awesome work in testing and captured them in the [PR description](https://github.com/alphagov/govuk-frontend/pull/2257). As this is a complex component with multiple interactions and there have been recent changes to the PR, we should carry out accessibility checks on our end as well to ensure that the component works correctly in our supported accessibility tools.

## Who needs to know about this

Devs, GOV.UK Accessibility team devs

## Done when

- [x] We've tested in our [supported assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#which-assistive-technologies-to-test-with)
- [x] ~We've arranged for testing with Dragon~ [Card for accessibility clinic](https://github.com/alphagov/govuk-frontend/issues/2295)
- [x] We've discussed potential issues with GOV.UK Accessibility team
- [x] We've decided if potential issues should be taken to the Accessibility clinic

**Comment by hannalaakso** [comment]
Doc to capture testing results: https://docs.google.com/spreadsheets/d/1HW1g67Cti5TE0Wz5DHzsWUL4uXI09iq5FshIu0-qZmg/edit#gid=0

**Comment by hannalaakso** [comment]
Re-tested following changes in PR: https://docs.google.com/spreadsheets/d/1HW1g67Cti5TE0Wz5DHzsWUL4uXI09iq5FshIu0-qZmg/edit#gid=1333924045

**Comment by hannalaakso** [comment]
Now working in this sheet of the doc: https://docs.google.com/spreadsheets/d/1HW1g67Cti5TE0Wz5DHzsWUL4uXI09iq5FshIu0-qZmg/edit#gid=1710385935
